cliqz/index.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
-import {AppRegistry, StyleSheet, Text, View, DeviceEventEmitter, } from 'react-native';
+import {AppRegistry, StyleSheet, Text, View, DeviceEventEmitter, NativeModules } from 'react-native';
 import SearchUI from 'browser-core/build/modules/mobile-cards-vertical/SearchUI';
```

Comment:
We should import both `mobile-cards` and `mobile-cards-vertical` and have an if condition in the render method, that reads a configuration and decides on the style it should render